autoload/ctrlp/mrufiles.vim

```diff
@@ -56,16 +57,22 @@ fu! s:reformat(mrufs, ...)
 endf
 
 fu! s:record(bufnr)
-	if s:exclnomod && &l:modifiable | retu | en
-  if s:exclnomod && !&l:modifiable | retu | en
```

Comment:
Did those 2 lines ever work as intended? I'm confused as to what they were supposed to accomplish.